experimental/regression_suite/shark-test-suite-models/sdxl/test_unet.py

```diff
@@ -218,7 +219,7 @@ def SDXL_PUNET_INT8_FP8_OUT(
 
 INT8_PUNET_FLAGS = [
     f"--iree-codegen-transform-dialect-library={iree_test_path_extension}/attention_and_matmul_spec_punet.mlir",
-    "--iree-preprocessing-pass-pipeline=builtin.module(util.func(iree-global-opt-raise-special-ops, iree-flow-canonicalize), iree-preprocessing-transpose-convolution-pipeline, iree-preprocessing-pad-to-intrinsics, util.func(iree-preprocessing-generalize-linalg-matmul-experimental))",
+    "--iree-preprocessing-pass-pipeline=builtin.module(util.func(iree-global-opt-raise-special-ops, iree-flow-canonicalize), iree-preprocessing-transpose-convolution-pipeline, iree-preprocessing-pad-to-intrinsics)",
```

Comment:
Isnt the `iree-preprocessing-transpose-convolution-pipeline` the one that is converting from NCHW to NHWC convolutions?